src/ui/components/DialogErrorWithTabs/DialogErrorWithTabs.scss

```diff
@@ -1,20 +1,33 @@
 .dl-dialog-error {
+    $contencDesktopMaxWidth: 630px;
+    $contentSmallScreenWidth: 300px;
     // TODO: this is an private api and may silently changed. We should use local styles
     --yc-tabs-space-between: 20px;
 
     $class: &;
 
+    width: 100%;
+    max-width: $contencDesktopMaxWidth;
+
     &__content {
-        width: 630px;
-        @media (min-width: 320px) and (max-width: 480px) {
-            width: 300px;
+        width: $contencDesktopMaxWidth;
+        @media (max-width: 480px) {
+            width: $contentSmallScreenWidth;
         }
         max-height: 80vh;
         height: 500px;
         display: flex;
         flex-direction: column;
     }
 
+    &__header {
```

Comment:
Why do you use overflow hidden with ellipsis for header? 
Firstly, we have rule that if we use ellipsis than we must add full text as a title to the same element.
Secondly, why can't we just display full caption text? Like this: 
<img width="666" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/98329176/ca678979-c46e-460c-a5c7-35061fee90ed">